website/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Custom Webpack Config"
+---
+
+<Subtitle>
+The utilities allows you create a custom webpack configuration for a Merchant Center Application.
+</Subtitle> 
+
+# Installation
+
+```
+yarn add @commercetools-frontend/mc-scripts
+
+# or
+
+npm --save install @commercetools-frontend/mc-scripts
+```
+
+# Usage
+To create a custom webpack configuration for a Merchant Center application, you can use `createWebpackConfigForProduction` for production and `createWebpackConfigForDevelopment` for development.
+These are factory functions used to create a default webpack config. They both accept some optional arguments.
+
+## Production
+For production mode, you should create a webpack.config.prod.(cjs, js) file and use the function `createWebpackConfigForProduction` to create a default webpack config.
```

Comment:
Maybe we want to highlight the file name too. Just a thought. What do you think?

```suggestion
For production mode, you should create a `webpack.config.prod.(cjs, js)`file and use the function `createWebpackConfigForProduction` to create a default webpack config.
```

Reply:
Updated [34feac4](https://github.com/commercetools/merchant-center-application-kit/pull/2726/commits/34feac438079f8a73467cd11f8b54646bd207d3c)